src/ember-app.js

```diff
@@ -450,6 +450,8 @@ function createShoebox(doc, fastbootInfo) {
 
     let value = shoebox[key];
     let textValue = JSON.stringify(value);
+    if (textValue === undefined) { continue; }
```

Comment:
@kellyselden your use-case seems fine to me. The part the confused me is why one would log this?